internal/pkg/agent/application/monitoring/process.go

```diff
@@ -61,3 +93,76 @@ func processHandler(coord *coordinator.Coordinator, statsHandler func(http.Respo
 		return errorWithStatus(http.StatusNotFound, fmt.Errorf("matching component %v not found", id))
 	}
 }
+
+func isProcessRedirectable(processName string) bool {
+	processNameLower := strings.ToLower(processName)
+	for _, prefix := range redirectableProcesses {
+		if strings.HasPrefix(processNameLower, prefix) {
+			return true
+		}
```

Comment:
[Question]
So the process name might be more than just the name in `redirectableProcesses`? Something like `apm-server-whatever`, and it's still be valid? If not, I'd say a more strict comparison would be better.

Reply:
yes this is what is required by cloud